sdk/communication/azure-communication-email/azure/communication/email/_generated/aio/operations/_patch.py

```diff
@@ -6,59 +6,71 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import Any, IO, Union
-from ._email_operations import EmailOperations as EmailOperationsGenerated
-from ... import models as _models
+import sys
+from typing import Any, IO, List, Union
+from ._operations import EmailOperations as EmailOperationsGenerated
+
+if sys.version_info >= (3, 9):
+    from collections.abc import MutableMapping
+else:
+    from typing import MutableMapping  # type: ignore  # pylint: disable=ungrouped-imports
+JSON = MutableMapping[str, Any]  # pylint: disable=unsubscriptable-object
 
 class EmailOperations(EmailOperationsGenerated):
 
-    def __return_message_id(self, pipeline_response, _, response_headers):
+    def _return_message_id(self, pipeline_response, _, response_headers):
         return response_headers['x-ms-request-id']
 
-    async def send(
+    async def send(  # pylint: disable=inconsistent-return-statements
```

Comment:
I'm not sure if you also need to patch the generated overloads - though maybe it won't matter given that you have wrapped the generated client.